corehq/apps/importer/util.py

```diff
@@ -455,11 +455,10 @@ def get_from_group(name):
         return getattr(group, 'get_id', None)
 
     def get_from_location(name):
-        for filter_ in [{'site_code': name}, {'name__iexact': name}]:
-            try:
-                return SQLLocation.objects.get(domain=domain, **filter_).location_id
-            except SQLLocation.DoesNotExist:
-                pass
```

Comment:
This isn't related to this PR at all - I was just moving this other thing I did which turns user input into a location into a model manager method so the logic is consistent.